src/components/Dropdowns/AccessibleDropdown.tsx

```diff
@@ -0,0 +1,240 @@
+import { Button, ButtonProps, Dropdown, theme, Tooltip } from "antd";
+import React, { ReactElement, ReactNode, useEffect, useRef, useState } from "react";
+import styled, { css } from "styled-components";
+
+import { DropdownSVG } from "../../assets";
+import { VisuallyHidden } from "../../styles/utils";
+
+type AccessibleDropdownProps = {
+  /** Text label to include with the dropdown. If null or undefined, hides the label. */
+  label?: string | null;
+
+  /** Contents to be rendered inside the dropdown. */
+  dropdownContent: ReactElement;
+  renderDropdownContent?: ((content: ReactElement) => ReactElement) | null;
+  disabled?: boolean;
+  /**
+   * The type of button to render for the dropdown. See Antd's button types:
+   * https://ant.design/components/button#components-button-demo-basic
+   */
+  buttonType?: ButtonProps["type"] | "outlined";
```

Comment:
Nit: looks like you could `import { ButtonType } from "antd/es/button"` to replace the `ButtonProps` import and save an index. 100% a matter of your preference though.

Reply:
I'm going to leave it as-is for now, since I slightly prefer not reaching into Ant's internals. Thank you for the comment though!